integration_tests/e2e/activities/editActivitySchedulePrisonRegime_2weekly.cy.ts

```diff
@@ -0,0 +1,143 @@
+
+    daysAndTimesPage.checkboxes().find('input[value="thursday"]').uncheck()
+    daysAndTimesPage.updateButton()
+
+    Page.verifyOnPage(ViewActivityPage)
+    viewActivityPage.assertNotificationContents(
+      'Activity updated',
+      `You've updated the daily schedule for English level 1`,
+    )
+  })
+  it('changing days/sessions if using prison regime times - adding a day', () => {
+    cy.visit('/activities/view/4')
+    const viewActivityPage = Page.verifyOnPage(ViewActivityPage)
+    viewActivityPage.changeScheduleLink().last().click()
+    const customTimesChangeOptionPage = Page.verifyOnPage(CustomTimesChangeOptionPage)
+    customTimesChangeOptionPage.changeDaysAndSessions('Days and sessions when this activity runs')
+    customTimesChangeOptionPage.continue()
+    const daysAndTimesPage = Page.verifyOnPage(DaysAndTimesPage)
+    daysAndTimesPage.title().contains('Week 2 of 2')
+    daysAndTimesPage.checkboxes().find('input[value="monday"]').should('be.checked')
+    daysAndTimesPage.getInputById('timeSlotsMonday-2').should('be.checked')
+    daysAndTimesPage.checkboxes().find('input[value="tuesday"]').should('be.checked')
+    daysAndTimesPage.getInputById('timeSlotsTuesday-2').should('be.checked')
+    daysAndTimesPage.checkboxes().find('input[value="wednesday"]').should('be.checked')
+    daysAndTimesPage.getInputById('timeSlotsWednesday-2').should('be.checked')
+
+    daysAndTimesPage.getInputById('timeSlotsWednesday').check()
+    daysAndTimesPage.updateButton()
+
+    Page.verifyOnPage(ViewActivityPage)
+    viewActivityPage.assertNotificationContents(
+      'Activity updated',
+      `You've updated the daily schedule for English level 1`,
+    )
+  })
+  it('changing times if currently using regime times', () => {
+    cy.visit('/activities/view/4')
+    const viewActivityPage = Page.verifyOnPage(ViewActivityPage)
+    viewActivityPage.changeScheduleLink().first().click()
+    const customTimesChangeOptionPage = Page.verifyOnPage(CustomTimesChangeOptionPage)
+    customTimesChangeOptionPage.changeDaysAndSessions('Activity start and end times')
+    customTimesChangeOptionPage.continue()
+    const sessionTimesPage = Page.verifyOnPage(SessionTimesPage)
+
+    sessionTimesPage.checkTableRow('1', 'Monday', 'AM', 0, '08', '30', '11', '45')
+    sessionTimesPage.checkTableRow('1', 'Tuesday', 'AM', 4, '08', '30', '11', '45')
+    sessionTimesPage.checkTableRow('1', 'Wednesday', 'AM', 8, '08', '30', '11', '45')
+    sessionTimesPage.checkTableRow('1', 'Thursday', 'AM', 12, '08', '30', '11', '45')
+    sessionTimesPage.checkTableRow('2', 'Monday', 'PM', 16, '13', '45', '16', '45')
+    sessionTimesPage.checkTableRow('2', 'Tuesday', 'PM', 20, '13', '45', '16', '45')
+    sessionTimesPage.checkTableRow('2', 'Wednesday', 'PM', 24, '13', '45', '16', '45')
+
+    sessionTimesPage.selectStartTime(10, 45, '1', 'MONDAY', 'AM')
+    sessionTimesPage.continue()
+  })
```

Comment:
Should this, verify the next page in the journey?

Reply:
👍